docs/helm/helm.md

```diff
@@ -38,22 +38,76 @@ spec:
       enabled: true
 ```
 
+Another example is deploying [cert-manager](https://cert-manager.io/) with authentication:
```

Comment:
This is a great example, but I think we should replace cert-manager with a more contrived example that is clearly pulling from a private chart repo, lest someone try to copy-paste this example to deploy cert-manager and open an issue complain that it's broken due to bad creds and CA cert.